docs/basics/userguide/export_training_data.md

```diff
@@ -0,0 +1,162 @@
+[INFO] 16:01: [PPO[worker: 1]] | max_global_step = 51200 | time/iterations = 24 | rollout/ep_rew_mean = 361.78 | rollout/ep_len_mean = 361.78 | time/fps = 325 | time/time_elapsed = 151 | time/total_timesteps = 49152 | train/learning_rate = 0.0003 | train/entropy_loss = -0.4632605144754052 | train/policy_gradient_loss = -0.0030253833654569464 | train/value_loss = 0.04603174216354091 | train/approx_kl = 0.005220792256295681 | train/clip_fraction = 0.06181640625 | train/loss = 0.01565437763929367 | train/explained_variance = -0.0232236385345459 | train/n_updates = 230 | train/clip_range = 0.2 |
+[INFO] 16:01: [PPO[worker: 2]] | max_global_step = 51200 | time/iterations = 24 | rollout/ep_rew_mean = 383.75 | rollout/ep_len_mean = 383.75 | time/fps = 323 | time/time_elapsed = 151 | time/total_timesteps = 49152 | train/learning_rate = 0.0003 | train/entropy_loss = -0.49069994343444706 | train/policy_gradient_loss = -0.0018766895205772015 | train/value_loss = 0.03171468693126371 | train/approx_kl = 0.005815165117383003 | train/clip_fraction = 0.061181640625 | train/loss = 0.0051743886433541775 | train/explained_variance = 0.12434303760528564 | train/n_updates = 230 | train/clip_range = 0.2 |
+[INFO] 16:01: [PPO[worker: 0]] | max_global_step = 51200 | time/iterations = 24 | rollout/ep_rew_mean = 370.78 | rollout/ep_len_mean = 370.78 | time/fps = 322 | time/time_elapsed = 152 | time/total_timesteps = 49152 | train/learning_rate = 0.0003 | train/entropy_loss = -0.49728800179436805 | train/policy_gradient_loss = -0.002779247868602397 | train/value_loss = 0.04587990254440229 | train/approx_kl = 0.005585251376032829 | train/clip_fraction = 0.0578125 | train/loss = 0.01021644752472639 | train/explained_variance = 0.028099477291107178 | train/n_updates = 230 | train/clip_range = 0.2 |
+[INFO] 16:01: ... trained!
+[INFO] 16:01: Saved ExperimentManager(PPO) using pickle.
+[INFO] 16:01: The ExperimentManager was saved in : 'rlberry_data/temp/manager_data/PPO_2024-06-28_15-58-48_4fc693bc/manager_obj.pickle'
+Backend tkagg is interactive backend. Turning interactive mode on.
+```
+
+
+Instead using {mod}`rlberry.manager.plot_writer_data` as in the [visualization page](visu_userguide), you can extract the information into dataframe to use it with other tools.
+To get these data, use the {mod}`rlberry.manager.read_writer_data` function.
+
+```python
+df = read_writer_data([manager])
+```
+
+Then, you can use your own tools to display whatever you like.
+
+
+In this example, it will be the rewards through the training of the 3 `fit` with matplotlib, but up to you display other information, use other tools, etc.
+
+<span>&#9888;</span> Of course, the information contained in the writer depends on how it has been configured and what the agent has recorded in it.<span>&#9888;</span>
```

Comment:
Maybe there could be an explanation of the writer somewhere: explain the system of tags, value, global_step... Maybe this could be in this section of the user-guide.

Reply:
All writers have the same operating mode and the same columns?